modules/aws_account_logging/cloudtrail/.gitignore

```diff
@@ -0,0 +1,9 @@
+*.dist-info
+*.egg-info
+certifi
+elasticsearch
+requests
+s3_log_parser
+ua_parser
+urllib3
+user_agents
```

Comment:
Probably worth moving this to the main gitignore.

Reply:
Added to main gitignore but I’m not removing it as this should be a vendored module later on and I want to keep it the same everywhere for now.